backend/routes/featureFlag.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const FeatureFlagModel = require('../models/FeatureFlag.js');
+const router = express.Router();
+
+// Save the data in the DB
+router.post('/', (req,res) => {
+    const feature = new FeatureFlagModel(req.body);
+    feature.save(); 
+});
```

Comment:
Moving the function to a controller folder would be a good idea. And also returning 500 status code response if there is any unexpected errors.


```js
// controller/featureFlag.js
const addFeatureFlag = (req,res) => {
    try {
      const feature = new FeatureFlagModel(req.body);
      feature.save(); 
    } catch(err) {
      // returns 500 
    }
}

// routes/featureFlag.js
router.post('/',  addFeatureFlag);
```

Reply:
Are you suggesting moving just the logic or renaming the folder to `controller`